app/models/concerns/order.rb

```diff
@@ -0,0 +1,48 @@
+module Order
```

Comment:
How about `Orderable` ?

I prefer `-able` like name for module because modules extract behavior.

https://docs.ruby-lang.org/ja/2.3.0/class/Enumerable.html